package-lock.json

```diff
@@ -230,6 +230,7 @@
           "resolved": "https://registry.npmjs.org/gaxios/-/gaxios-1.8.4.tgz",
           "integrity": "sha512-BoENMnu1Gav18HcpV9IleMPZ9exM+AvUjrAOV4Mzs/vfz2Lu/ABv451iEXByKiMPn2M140uul1txXCg83sAENw==",
           "dev": true,
+          "optional": true,
```

Comment:
Gotta get rid of all this nonsense.

Reply:
@yuchenshi I can't tell if I am wrong or `master` is wrong, I am using `npm` 6.9.2

Reply:
I think master is wrong since https://github.com/firebase/firebase-tools/pull/1433/files

Reply:
Ah thank you!